platform/commonUI/general/res/sass/overlay/_overlay.scss

```diff
@@ -19,8 +19,17 @@
  * this source code distribution or the Licensing information page available
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
+
 .overlay {
     font-size: 90%;
+    &.delayEntry100ms {
+        @include keyframes(fadeInFromNone) {
+            0% { display: none; opacity: 0; }
+            100% { display: block; opacity: 1; }
+        }
+        @include animation-delay($delayEntryMs);
+        @include animation(fadeInFromNone $durEntryMs ease-in);
```

Comment:
@akhenry I just pushed an update to shorten the duration to 50ms which was requested a few comments back. Are you wanting that to be shorter or zero? I'm good either way.